network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -43,7 +43,7 @@ def generate_header_field():
 
 def generate_image_field():
     images = Image.objects.all()
-    image_idx = fake.unix_time() % images.count()
+    image_idx = randint(0, images.count() - 1)
```

Comment:
it's interesting to see that python's randint yields `a <= N <= b` rather than `a <= N < b`, but that's where the `-1` comes from.

Reply:
Did you check [`random.choice()`](https://docs.python.org/3/library/random.html#random.choice)?

Reply:
oh cool, I had not!

Reply:
nice, switched!